site/docs/tutorial/java.md

```diff
@@ -47,6 +48,22 @@ In this tutorial you'll learn how to:
 To prepare for the tutorial, first [Install Bazel](../install.md) if
 you don't have it installed already.
 
+### Install the JDK
+
+1.  Install Java 8 JDK.
+
+2.  Set the JAVA\_HOME environment variable to point to the JDK.
+    *   On Linux/macOS:
+
+            export JAVA_HOME="$(dirname $(dirname $(realpath $(which javac))))"
```

Comment:
add triple backticks

Reply:
add `$` prompt?

Reply:
4-spaces indentation has the same effect. I checked that this renders correctly using `scripts/serve-docs.sh`.

Reply:
I'd rather not add `$` so it's easier to copy-paste the snippet.